hub/core/meta/tensor_meta.py

```diff
@@ -233,9 +233,12 @@ def _format_values(htype_overwrite: dict):
 
     if htype_overwrite["dtype"] is not None:
         htype_overwrite["dtype"] = np.dtype(htype_overwrite["dtype"]).name
+    if htype_overwrite.get("sample_compression") == "jpg":
+        htype_overwrite["sample_compression"] = "jpeg"
```

Comment:
can you have a dictionary in constants.py for "compression_aliases". key = alias, value = actual.

"jpg": "jpeg"

then we can add more later

Reply:
@mccrearyd I thought about doing that, but it seems way overkill. This can be refactored out if it ever comes up again.

Reply:
not really overkill, it's just a dictionary the same lines of code. but sure i will approve it